### ISSUE TITLE
Future plans on the lib

### ISSUE THREAD

**Issue by senpos** [python-pendulum/pendulum#354]
Hi.

Do you have any plans on improving pendulum? At least by merging viable PRs.
Or maybe there's a chance we'll have collaborator who is interested in maintaining it?

I love pendulum and was really happy about 1.x version, but after new major release several things are broken and lib status is neither alpha nor stable. It's somewhere in between. :)

Please, give us some feedback about pendulum's future.

**Comment by tuukkamustonen** [comment]
Also curious about the future of the library. I originally betted on pendulum to win the race over [arrow](https://github.com/crsmithdev/arrow), but looking at the projects now, `pendulum` seems stale (few commits during last ~1.5 years, tickets without activity, lot of open PRs) and arrow is enjoying sunshine (constant flow of commits, 3x contributors, less tickets open, ~0 PRs open).

This is, of course, not criticism on author, who has done awesome job providing the library. But what sort of future does @sdispater see for the library?

**Comment by sdispater** [comment]
The library is not abandoned in any way but [poetry](https://github.com/python-poetry/poetry) is taking most of my limited time currently.

I also have some new features in mind that I will try to implement in the coming weeks.

That being said, since I can't give my full attention to the library, I will try to onboard new maintainers to help with its maintenance and evolution.

So if anyone is interested in helping out with that, please let me know :-)

**Comment by tpakhoa1996** [comment]
Will there be an update on this repo?

**Comment by tpakhoa1996** [comment]
I think the library is going in the right direction but it needs to be more active. If finance is a problem, how about creating Patreon and discord?

**Comment by sdispater** [comment]
@karolzlot The next major version is being worked on (see #569) and will feature a redesign and a lot of improvements.

While Arrow has indeed picked up the pace around DST handling and parsing, Pendulum still provides a more intuitive API and is still a drop-in replacement for, and with a lot of improvements over, the standard library, which Arrow is not.

**Comment by spacemanspiff2007** [comment]
There are critical issues e.g. where users have been unable to install pendulum (see #504) which have not been fixed since in over a year.

I understand that time is limited but when there is nobody else that can/does work on pendulum it is indeed abandon ware.
There are definitely some people who would want to join in as maintainers, so maybe that would get development started again.

**Comment by sdispater** [comment]
The first prerelease of the next version (`3.0`) is now available for testing: https://github.com/sdispater/pendulum/releases/tag/3.0.0a1

**Comment by anderssonjohan** [comment]
@sdispater Great job! We're currently moving everything to latest poetry from 1.0.10 and python 3.11 but are struggling a bit without prebuilt 3.11 wheels for pendulum 2.x. Is there a chance you can backport the build or just do a manual upload of a 3.11 wheel for 2.1.2? 🙏 

We would rather not publish our own pendulum 2.1.2 wheel on a private index, because I suspect we'll get an "Invalid hash" issue with poetry the day an official wheel is published. Also, it would save build minutes on a "global" scale if there were official 3.11 wheels for the latest release now that many are moving to that version. :)

I fully understand if you are busy with other things, no worries if you don't have the time needed to get a release out. 🙏

**Comment by vbdevpro** [comment]
> @sdispater Great job! We're currently moving everything to latest poetry from 1.0.10 and python 3.11 but are struggling a bit without prebuilt 3.11 wheels for pendulum 2.x. Is there a chance you can backport the build or just do a manual upload of a 3.11 wheel for 2.1.2? 🙏
> 
> We would rather not publish our own pendulum 2.1.2 wheel on a private index, because I suspect we'll get an "Invalid hash" issue with poetry the day an official wheel is published. Also, it would save build minutes on a "global" scale if there were official 3.11 wheels for the latest release now that many are moving to that version. :)
> 
> I fully understand if you are busy with other things, no worries if you don't have the time needed to get a release out. 🙏

Agreed. Not having 3.11 wheel completely breaks the usage of this project for prefect users on 3.11.

**Comment by ashb** [comment]
Closing this now as in #590 we have increased the number of maintainers and moved the package to an org 🎉
